easybuild/easyconfigs/p/PyGWAS/PyGWAS-1.2.0-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = "PythonPackage"
+
+name = 'PyGWAS'
+version = '1.2.0'
+
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/timeu/pygwas'
+description = """PyGWAS is a library for running Genome Wide Association studies."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['2fd4264b92697efab7a09f815f004015554b3e46c74c81d8ed1bec8d2f20f9c9']
+
+pythonversion = '2.7.11'
+hdf5ver = '1.8.16'
+versionsuffix = '-Python-%s' % pythonversion
```

Comment:
don't repeat `versionsuffix`, just once is enough (and keep it close/right below to the `version =` line above)